src/lib/index.js

```diff
@@ -1195,6 +1195,18 @@ lib.isHidden = function(gd) {
     return !display || display === 'none';
 };
 
+lib.strTranslate = function(x, y) {
+    return (x || y) ? 'translate(' + x + ',' + y + ')' : '';
+};
```

Comment:
Might be able to simplify even more with an optional 3rd argument which is the number of digits to round to - would remove the `Math.round` and `toFixed` you have in a few places.